src/components/Header/NetworkSelector.tsx

```diff
@@ -242,9 +242,9 @@ function Row({
 
 const getParsedChainId = (parsedQs?: ParsedQs) => {
```

Comment:
Can you make this a function, and explicitly type the return type? It looks like it's changed, and can now be `void | string`?

    function getChainIdFromParsedQs(parsedQs?: ParsedQs): string | undefined {